Tests/ArgumentParserUnitTests/ParsableArgumentsValidationTests.swift

```diff
@@ -167,6 +167,17 @@ final class ParsableArgumentsValidationTests: XCTestCase {
     var options: Options
   }
 
+  private struct L: ParsableArguments {
```

Comment:
Could you add a comment here that this is a compilation test validating that parentheses aren't required on the property wrappers? It looks unused otherwise.

Reply:
I'll add the comment later, want to get this merged before to deal w/ conflicts. 👍

Reply:
Thanks! I might also follow up with a commit to update the documentation to use this style.